### PR TITLE
Another pip install attempt

### PR DESCRIPTION
**Pull request by connorjward**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -176,43 +206,70 @@ endif()
 find_path(spatialindexheaders
           NAMES spatialindex/SpatialIndex.h
           REQUIRED
-          PATHS /usr/local/include /usr/include $ENV{VIRTUAL_ENV}/lib/python${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/site-packages/rtree/include
+          PATHS /usr/local/include /usr/include $ENV{VIRTUAL_ENV}/lib/python${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/site-packages/rtree/include spatialindexheaders
```
Comment (JDBetteridge) [inline CMakeLists.txt]:
```suggestion
          PATHS /usr/local/include /usr/include $ENV{VIRTUAL_ENV}/lib/python${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/site-packages/rtree/include
```
99% sure that is a typo

**Review by JDBetteridge — CHANGES_REQUESTED** [review]
Minor typo (I think...)

**Comment by connorjward** [comment]
> Minor typo (I think...)

Lets see what CI says. I agree it's almost definitely going to be fine. Assuming things pass may I hit merge?

**Review by JDBetteridge — APPROVED** [review]

**Comment by JDBetteridge** [comment]
Yupp, it won't break anything in its current state (the old install path is still supported)

**Pull request merged**